Makefile

```diff
@@ -0,0 +1,36 @@
+.PHONY: all build clean copy-src local requirements setup test up update-dependencies
+
+CKAN_HOME := /srv/app
+
+all: build
+
+build:
+	docker-compose build
+
+clean:
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml down -v --remove-orphans
+
+copy-src:
+	docker cp catalog-app_ckan_1:$(CKAN_HOME)/src .
+
+dev:
+	docker-compose -f docker-compose.dev.yml build
+	docker-compose -f docker-compose.dev.yml up
+
+debug:
+	docker-compose -f docker-compose.dev.yml build
+	docker-compose -f docker-compose.dev.yml run --service-ports ckan-dev
+
+requirements:
+	docker-compose run --rm -T ckan pip --quiet freeze > requirements-freeze.txt
+
+test:
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.test.yml build
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.test.yml up --abort-on-container-exit test
+
+update-dependencies:
+	docker-compose run --rm -T ckan pip --quiet freeze > requirements-freeze.txt
+	docker-compose run --rm -T ckan pip install -r requirements-freeze.txt
```

Comment:
I'm not sure this will work .... the `requirements-freeze.txt` file is not actually in the container, as far as I can tell.

Reply:
Fixed